### PR TITLE
Added upward navigation for CityMapsActivity

### PR DESCRIPTION
**Pull request by Florence-Njeri**
## Upwards Navigation
I have added up navigation to the CityMapsActivity

### CODE REVIEW

File: Android/build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:3.4.0'
```
Comment (Swati4star) [inline Android/build.gradle]: why this change?
Is there any reason for downgrading the version?

**Comment by Florence-Njeri** [comment]
Ooh sorry its because I have not updated to 3.5 on my side yet

Sent from Mail for Windows 10

File: Android/build.gradle (hunk 1 above)
Comment (Swati4star) [reply Android/build.gradle]: @Florence-Njeri Revert this change. Then only, I'll be able to approve and merge it.

**Comment by prabhakar267** [comment]
@Florence-Njeri Can you update to the project's version and update your PR?

File: Android/app/src/main/java/io/github/project_travel_mate/destinations/description/CityMapActivity.java [hunk 2]
```diff
@@ -40,6 +42,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         ButterKnife.bind(this);
 
         final City mCity = (City) getIntent().getSerializableExtra(EXTRA_MESSAGE_CITY_OBJECT);
+        Objects.requireNonNull(getSupportActionBar()).setDisplayHomeAsUpEnabled(true);
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/destinations/description/CityMapActivity.java]: This change can cause app to crash: see this: https://github.com/project-travel-mate/Travel-Mate/pull/735/files

File: Android/build.gradle (hunk 1 above)
Comment (Swati4star) [reply Android/build.gradle]: Revert this change

File: Android/app/src/main/java/io/github/project_travel_mate/destinations/description/CityMapActivity.java (hunk 2 above)
Comment (Florence-Njeri) [reply Android/app/src/main/java/io/github/project_travel_mate/destinations/description/CityMapActivity.java]: I can see you fixed that . So whats the issue?

Comment (Swati4star) [reply Android/app/src/main/java/io/github/project_travel_mate/destinations/description/CityMapActivity.java]: that has been fixed for FunFactsActivity.java 
Do similar thing here also

**Comment by Swati4star** [comment]
@Florence-Njeri Closing this PR because of inactivity. Feel free to reopen it if you continue to workn on it
